packages/ethereum-storage/src/ethereum-storage.ts

```diff
@@ -71,8 +71,8 @@ export default class EthereumStorage implements StorageTypes.IStorage {
    */
   public constructor(
     externalBufferUrl: string,
-    ipfsGatewayConnection?: StorageTypes.IIpfsGatewayConnection,
-    web3Connection?: StorageTypes.IWeb3Connection,
+    ipfsGatewayConnection: StorageTypes.IIpfsGatewayConnection,
```

Comment:
No risk of breaking backward compatibility ?

Reply:
ethereum storage is only used by the node, so no